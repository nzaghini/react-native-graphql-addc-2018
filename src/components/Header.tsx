import React from "react";
import { View, Text, StyleSheet } from "react-native";

const Header = () => {
    const { container, title } = styles;
    return (
        <View style={container}>
            <Text style={title}>Movies</Text> 
        </View>
    );
};

const styles = StyleSheet.create({
    container: {
      backgroundColor: "#F8F8F8",
      justifyContent: "center", // vertical
      alignItems: "center", // horizontal
      height: 60,
      shadowColor: "black",
      shadowOffset: { width: 0, height: 1 },
      shadowOpacity: 0.2,
    },
    title: {
      fontSize: 18,
    },
});

export default Header;
