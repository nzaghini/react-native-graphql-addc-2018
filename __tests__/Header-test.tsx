import React from "react";
import renderer from "react-test-renderer";
import Header from "../src/components/Header";

describe("<Header />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
