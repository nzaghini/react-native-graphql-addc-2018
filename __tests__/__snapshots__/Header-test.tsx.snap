// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F8F8F8",
      "height": 60,
      "justifyContent": "center",
      "shadowColor": "black",
      "shadowOffset": Object {
        "height": 1,
        "width": 0,
      },
      "shadowOpacity": 0.2,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 18,
      }
    }
  >
    Movies
  </Text>
</View>
`;
